samples/word/10-content-controls/get-change-tracking-states.yaml

```diff
@@ -0,0 +1,174 @@
+order: 9
+id: word-content-controls-get-change-tracking-states
+name: Get change tracking states of content controls
+description: Inserts and deletes content controls then retrieves their change tracking state.
```

Comment:
We already have samples that focus on adding and removing content controls. Should we change this description to focus on the tracking state?